tools/keystore_generator.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+
+CA_CERT_PATH=$CA_CERT_PATH
+CA_KEY_PATH=$CA_KEY_PATH
+KEYSTORE_PASS=$KEYSTORE_PASS
+
+if [ ! -e "$CA_CERT_PATH" ] || [ ! -e "$CA_KEY_PATH" ]; then
+  echo "CA does not exist, please provide the corrrect paths in CA_CERT_PATH and CA_KEY_PATH"
+  exit 1
+fi
+
+if [ -z "$KEYSTORE_PASS" ]; then
+  echo "Please provide a keystore password in KEYSTORE_PASS"
+  exit 1
+fi
+
+# Generate truststore containing CA
+keytool -keystore ./kafka.truststore.jks \
```

Comment:
Where does `keytool` come from?

Reply:
`keytool` comes with any Java distribution. I guess this means that the script assumes the user has Java on their system. Would this be an issue?